docs/getting-started/fullstack-example.md

```diff
@@ -0,0 +1,79 @@
+---
+id: fullstack-example
+title: Full Stack Example
+
+---
+
+This example will walk through putting together a full stack app on KintoHub from the ground up and will take about 15 minutes.
+If you are interested in a quick single service, check out our list of [examples](examples/nodejs/nodejs-example).
+
+This example will deploy a **Todo List App** using `React`, `Hasura` and `Postgres` all within KintoHub's free tier.
+
+## Create a KintoHub Account
+
+If you have not created a KintoHub account yet, please to create one [here](https://app.kintohub.com).
+
+## Deploy a Postgres Server
+
+To store our data, we will need a Postgres database.
+KintoHub provides a dedicated Postgres server with 1 Gb of storage.
+
+1. Click the **Create Service** At the top right of your environment
+2. Click the **From Catalog** tab
+3. Click **PostgreSQL**
+4. Enter or generate your `Username`, `Password`, and `Root Password`.
+5. Click **Deploy** At the top right.
+
+This process will take about **1 Minute** to deploy.
+Once the **Status** has changed from `Deploying` to `Live Version`, click on the **Access** tab at the top center of the page.
+
+Copy the **Connection String (Admin)** and paste it somewhere for setting up Hasura.
```

Comment:
keep it somewhere?

Reply:
@Disturbing  Yeah I guess it's just the term, cuz it makes me think that I need to paste it somewhere on kintohub.